### DISCUSSION TITLE
Welcome to Mi-Create Discussions!

### DISCUSSION THREAD

**Discussion by ooflet** [ooflet/Mi-Create#4] [Announcements/Объявления/公告]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by vtungabc** [comment]
How to make animation watch face ? I want to use animations in my watch face !

**Reply by ooflet** [reply]
Use the guide on EasyFace, it works for Mi Create as well https://github.com/m0tral/EasyFace/wiki/Animations. You need to setup an image list, set the amount of frames you have in your animation with image count property and fill each image field with the frame. The index is not important here.

**Comment by ObiJuanKenobe1** [comment]
hi all,
I´ve just achived a miband 10, I used to create mi own watchfaces with **mi create**, but it do not support mi band 10 yet. Are you working on a new version including this new band?
there is a new version of easyface that supports miband 10 but I do prefer mi create by far.
thak you

**Reply by ooflet** [reply]
You can update the compiler and device list with the current version of Mi Create in Settings > Three Dots menu on the top left > Update Compiler from EasyFace. This will add the Mi Band 10 to the device list so you can make and compile for that device.

**Reply by ObiJuanKenobe1** [reply]
thank you for your answer but I´m afraid I´m not able to update the compiler. There is something I´m doing wrong...I follow your instruction and micreate opens an explorer window loking for an input path. I input the folder where easyface file "complier.exe" is in my PC and micreate doesn´t recornise the file extension....
This is the message I receive:
<img width="377" height="124" alt="Captura de pantalla 2025-07-15 154431" src="https://github.com/user-attachments/assets/f1001b8c-683f-476c-a1ea-c06f31a66417" />

Can you help me?
thanks in advance

**Reply by ooflet** [reply]
No worries, I've actually just released a proper version update with many long overdue bug fixes and the new compiler + device, give it a try :smile: https://github.com/ooflet/Mi-Create/releases/tag/v1.1.1

**Comment by ObiJuanKenobe1** [comment]
Yes Sir.
It looks great!😍​
👏​👏​👏​👏​

thank you so much

🙌​​🙌​​🙌​

________________________________
De: ooflet ***@***.***>
Enviado: Miércoles, 16 de Julio de 2025 14:12
Para: ooflet/Mi-Create ***@***.***>
CC: ObiJuanKenobe1 ***@***.***>; Comment ***@***.***>
Asunto: Re: [ooflet/Mi-Create] Welcome to Mi-Create Discussions! (Discussion #4)


No worries, I've actually just released a proper version update with many long overdue bug fixes and the new compiler + device, give it a try 😄 https://github.com/ooflet/Mi-Create/releases/tag/v1.1.1
